### PR TITLE
docs: Adding info re container signatures

### LINKED ISSUE #1337: `skopeo copy` to local file-system doesn't appear to store signatures
Using `skopeo` `1.3.1-dev` on Ubuntu `20.04` or `1.3.0` on macOS `11.4`, I don't appear to be storing signatures when using `skopeo copy` to copy *from* Docker Hub *to* the local file-system.

In both cases, I'm running: -

`skopeo --debug copy --all docker://davidhay1969/hello_world_nginx_dct:latest dir:///tmp/hello_world_nginx_dct`

On the Ubuntu box, this returns: -

```text

DEBU[0000] overwriting existing container image directory "/tmp/hello_world_nginx_dct" 
DEBU[0000] Loading registries configuration "/etc/containers/registries.conf" 
DEBU[0000] Loading registries configuration "/etc/containers/registries.conf.d/000-shortnames.conf" 
DEBU[0000] Trying to access "docker.io/davidhay1969/hello_world_nginx_dct:latest" 
DEBU[0000] Trying to access "docker.io/davidhay1969/hello_world_nginx_dct:latest" 
DEBU[0000] Found credentials for docker.io in credential helper containers-auth.json 
DEBU[0000] Using registries.d directory /etc/containers/registries.d for sigstore configuration 
DEBU[0000]  Using "default-docker" configuration        
DEBU[0000]  No signature storage configuration found for docker.io/davidhay1969/hello_world_nginx_dct:latest, using built-in default file:///var/lib/containers/sigstore 
DEBU[0000] Looking for TLS certificates and private keys in /etc/docker/certs.d/docker.io 
DEBU[0000] GET https://registry-1.docker.io/v2/         
DEBU[0000] Ping https://registry-1.docker.io/v2/ status 401 
DEBU[0000] GET https://auth.docker.io/token?account=davidhay1969&scope=repository%3Adavidhay1969%2Fhello_world_nginx_dct%3Apull&service=registry.docker.io 
DEBU[0000] GET https://registry-1.docker.io/v2/davidhay1969/hello_world_nginx_dct/manifests/latest 
DEBU[0001] Content-Type from manifest GET is "application/vnd.docker.distribution.manifest.v2+json" 
DEBU[0001] Using blob info cache at /var/lib/containers/cache/blob-info-cache-v1.boltdb 
DEBU[0001] IsRunningImageAllowed for image docker:docker.io/davidhay1969/hello_world_nginx_dct:latest 
DEBU[0001]  Using default policy section                
DEBU[0001]  Requirement 0: allowed                      
DEBU[0001] Overall: allowed                             
Getting image source signatures
DEBU[0001] Reading /var/lib/containers/sigstore/davidhay1969/hello_world_nginx_dct@sha256=0de773124007c92b3bc8f00052a76d3568247e0806a2306bce12930e64d0c839/signature-1 
DEBU[0001] Downloading /v2/davidhay1969/hello_world_nginx_dct/blobs/sha256:7184c046fdf17da4c16ca482e5ede36e1f2d41ac8cea9c036e488fd149d6e8e7 
DEBU[0001] GET https://registry-1.docker.io/v2/davidhay1969/hello_world_nginx_dct/blobs/sha256:7184c046fdf17da4c16ca482e5ede36e1f2d41ac8cea9c036e488fd149d6e8e7 
DEBU[0001] Detected compression format gzip             
DEBU[0001] Using original blob without modification     
Copying blob 7184c046fdf1 done  
DEBU[0002] Downloading /v2/davidhay1969/hello_world_nginx_dct/blobs/sha256:405ecb930bd23ebd55e695079234dfdd38d15a9eeb102d1811ba249aa69824c1 
Copying blob 7184c046fdf1 done  
DEBU[0002] Detected compression format gzip             
Copying blob 7184c046fdf1 done  
Copying blob 405ecb930bd2 done  
Copying blob 7184c046fdf1 done  
Copying blob 405ecb930bd2 done  
Copying blob 7184c046fdf1 done  
Copying blob 405ecb930bd2 done  
Copying blob 7184c046fdf1 done  
Copying blob 405ecb930bd2 done  
Copying blob 7184c046fdf1 done  
Copying blob 405ecb930bd2 done  
Copying blob 7184c046fdf1 done  
Copying blob 405ecb930bd2 done  
Copying blob 7184c046fdf1 done  
Copying blob 405ecb930bd2 done  
Copying blob 7184c046fdf1 done  
Copying blob 405ecb930bd2 done  
Copying blob 7184c046fdf1 done  
Copying blob 405ecb930bd2 done  
Copying blob a076d626a918 done  
Copying blob e580469fb1c9 done  
Copying blob 97911c8d3734 done  
Copying blob 417b4b25b6fd done  
DEBU[0004] Downloading /v2/davidhay1969/hello_world_nginx_dct/blobs/sha256:7c64cf4d0e3dfde1067f35995eaf182f0d22d74f80b854f0e52e91c6b81e20f8 
DEBU[0004] GET https://registry-1.docker.io/v2/davidhay1969/hello_world_nginx_dct/blobs/sha256:7c64cf4d0e3dfde1067f35995eaf182f0d22d74f80b854f0e52e91c6b81e20f8 
DEBU[0005] No compression detected                      
DEBU[0005] Using original blob without modification     
Copying config 7c64cf4d0e done  
Writing manifest to image destination
Storing signatures
```

On the Mac, this returns: -

```text

DEBU[0000] overwriting existing container image directory "/private/tmp/hello_world_nginx_dct" 
DEBU[0000] Loading registries configuration "/usr/local/etc/containers/registries.conf" 
DEBU[0000] Trying to access "docker.io/davidhay1969/hello_world_nginx_dct:latest" 
DEBU[0000] Trying to access "docker.io/davidhay1969/hello_world_nginx_dct:latest" 
DEBU[0000] Found credentials for docker.io in credential helper containers-auth.json 
DEBU[0000] Using registries.d directory /usr/local/etc/containers/registries.d for sigstore configuration 
DEBU[0000]  Using "default-docker" configuration        
DEBU[0000]  No signature storage configuration found for docker.io/davidhay1969/hello_world_nginx_dct:latest, using built-in default file:///Users/hayd/.local/share/containers/sigstore 
DEBU[0000] Looking for TLS certificates and private keys in /etc/docker/certs.d/docker.io 
DEBU[0000] GET https://registry-1.docker.io/v2/         
DEBU[0000] Ping https://registry-1.docker.io/v2/ status 401 
DEBU[0000] GET https://auth.docker.io/token?account=davidhay1969&scope=repository%3Adavidhay1969%2Fhello_world_nginx_dct%3Apull&service=registry.docker.io 
DEBU[0001] GET https://registry-1.docker.io/v2/davidhay1969/hello_world_nginx_dct/manifests/latest 
DEBU[0001] Content-Type from manifest GET is "application/vnd.docker.distribution.manifest.v2+json" 
DEBU[0001] Using blob info cache at /Users/hayd/.local/share/containers/cache/blob-info-cache-v1.boltdb 
DEBU[0001] IsRunningImageAllowed for image docker:docker.io/davidhay1969/hello_world_nginx_dct:latest 
DEBU[0001]  Using default policy section                
DEBU[0001]  Requirement 0: allowed                      
DEBU[0001] Overall: allowed                             
Getting image source signatures
DEBU[0001] Reading /Users/hayd/.local/share/containers/sigstore/davidhay1969/hello_world_nginx_dct@sha256=0de773124007c92b3bc8f00052a76d3568247e0806a2306bce12930e64d0c839/signature-1 
DEBU[0001] Downloading /v2/davidhay1969/hello_world_nginx_dct/blobs/sha256:7184c046fdf17da4c16ca482e5ede36e1f2d41ac8cea9c036e488fd149d6e8e7 
DEBU[0001] GET https://registry-1.docker.io/v2/davidhay1969/hello_world_nginx_dct/blobs/sha256:7184c046fdf17da4c16ca482e5ede36e1f2d41ac8cea9c036e488fd149d6e8e7 
DEBU[0002] Detected compression format gzip             
DEBU[0002] Using original blob without modification     
Copying blob 7184c046fdf1 done  
DEBU[0009] Downloading /v2/davidhay1969/hello_world_nginx_dct/blobs/sha256:405ecb930bd23ebd55e695079234dfdd38d15a9eeb102d1811ba249aa69824c1 
Copying blob 7184c046fdf1 done  
DEBU[0009] Detected compression format gzip             
Copying blob 7184c046fdf1 done  
Copying blob 405ecb930bd2 done  
Copying blob 7184c046fdf1 done  
Copying blob 405ecb930bd2 done  
Copying blob 7184c046fdf1 done  
Copying blob 405ecb930bd2 done  
Copying blob 7184c046fdf1 done  
Copying blob 405ecb930bd2 done  
Copying blob 7184c046fdf1 done  
Copying blob 405ecb930bd2 done  
Copying blob 7184c046fdf1 done  
Copying blob 405ecb930bd2 done  
Copying blob 7184c046fdf1 done  
Copying blob 405ecb930bd2 done  
Copying blob 7184c046fdf1 done  
Copying blob 405ecb930bd2 done  
Copying blob 7184c046fdf1 done  
Copying blob 405ecb930bd2 done  
Copying blob a076d626a918 done  
Copying blob e580469fb1c9 done  
Copying blob 97911c8d3734 done  
Copying blob 417b4b25b6fd done  
DEBU[0026] Downloading /v2/davidhay1969/hello_world_nginx_dct/blobs/sha256:7c64cf4d0e3dfde1067f35995eaf182f0d22d74f80b854f0e52e91c6b81e20f8 
DEBU[0026] GET https://registry-1.docker.io/v2/davidhay1969/hello_world_nginx_dct/blobs/sha256:7c64cf4d0e3dfde1067f35995eaf182f0d22d74f80b854f0e52e91c6b81e20f8 
DEBU[0026] No compression detected                      
DEBU[0026] Using original blob without modification     
Copying config 7c64cf4d0e done  
Writing manifest to image destination
Storing signatures
```

However, in both cases, I don't appear to have anything stored in the signature store: -

**Ubuntu**

`ls -al /var/lib/containers/sigstore/`

```text
total 0
drwxr-xr-x 2 root root  6 Jun 16 06:40 .
drwxr-xr-x 5 root root 50 Jun 24 11:47 ..
```

**macOS**

`ls -al /Users/hayd/.local/share/containers/sigstore`

```text
total 0
drwxr-xr-x  2 hayd  staff   64 25 Jun 16:49 .
drwx------  4 hayd  staff  128 25 Jun 16:49 ..
```

( I even created the `sigstore` subdirectory under `~/.local/share/containers` but to no avail )

For the record, this image was built/pushed using Docker Content Trust enabled: -

```bash
export DOCKER_CONTENT_TRUST=1
DOCKER_CONTENT_TRUST_SERVER=https://docker.io:4443
docker build -t davidhay1969/hello_world_nginx_dct_june_2021:latest -f Dockerfile .
docker push davidhay1969/hello_world_nginx_dct_june_2021:latest
```

As ever, this is probably pilot error but ... 🤷

### PR DESCRIPTION
**Pull request by davidhay1969**
Updating docs to reflect container signatures

Fixes #1337

Signed-off-by: Dave Hay <david_hay@uk.ibm.com>

### CODE REVIEW

File: docs/skopeo-standalone-sign.1.md [hunk 1]
```diff
@@ -4,11 +4,10 @@
 skopeo\-standalone-sign - Debugging tool - Publish and sign an image in one step.
 
 ## SYNOPSIS
-**skopeo standalone-sign** [*options*] _manifest_ _docker-reference_ _key-fingerprint_
+**skopeo standalone-sign** [*options*] _manifest_ _docker-reference_ _key-fingerprint_ **--output**|**-o** _signature_
```
Comment (rhatdan) [inline docs/skopeo-standalone-sign.1.md:7]: The --output option is handled by [*option*].

We don't display options on the SYNOPSIS line.

Comment (mtrmac) [reply docs/skopeo-standalone-sign.1.md:7]: This one is mandatory, so the square-bracketed [option] is not appropriate.

**Review by mtrmac — APPROVED** [review]
LGTM (all earlier discussion in #1355 ).

File: docs/skopeo-standalone-sign.1.md (hunk 1 above)
Comment (mtrmac) [reply docs/skopeo-standalone-sign.1.md:7]: … and already in the repo per #1351, not really a part of this PR.

Comment (davidhay1969) [reply docs/skopeo-standalone-sign.1.md:7]: 😕 I don't think I changed that element .... looking back at a [prior commit](https://github.com/containers/skopeo/commit/22908fb3e8a17c42675a7471c9b365c507a79a3c#diff-eb57f4b9e43694badc7d10fdfa02d65094b136948eb433f003e9c168b0db0833) it looks like that was changed back in late June @rhatdan

**Comment by mtrmac** [comment]
@davidhay1969 we have added some validation in the mean time, let’s see how a rebased version fares with the tests: #1381.

**Comment by mtrmac** [comment]
Merged in #1381. Thanks again!

**Pull request closed without merging**